From: Jan Kara <jack@suse.cz>
Date: Thu, 26 Nov 2015 13:29:21 +0100
Subject: [PATCH 4/5] cgroup: Export cgroup_is_descendant()
Patch-mainline: Never, function moved in b11cfb5807e30
References: bsc#951265

CFQ IO scheduler will use cgroup_is_descendant() to decide whether one
queue can preempt another one.

Acked-by: Tejun Heo <tj@kernel.org>
Signed-off-by: Jan Kara <jack@suse.cz>
---
 kernel/cgroup.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/cgroup.c b/kernel/cgroup.c
index 470f6536b9e8..c4d74c10dfcb 100644
--- a/kernel/cgroup.c
+++ b/kernel/cgroup.c
@@ -483,6 +483,7 @@ bool cgroup_is_descendant(struct cgroup *cgrp, struct cgroup *ancestor)
 	}
 	return false;
 }
+EXPORT_SYMBOL_GPL(cgroup_is_descendant);
 
 static int notify_on_release(const struct cgroup *cgrp)
 {
-- 
2.6.2

